### PR TITLE
Added Ceganpost fcn

### PR DESCRIPTION
**Pull request by johnsnoa**
Added Ceganpost fcn, doubled msg count to trigger Peet, added experimental thing to try and deal with username changes for cegan

### CODE REVIEW

**Comment by johnsnoa** [comment]
The code here is more or less the same as Clayton's section, the only difference is in the new on_user_update(User, User) definition, which (from what I can tell) should work fine.

File: inferno_cop.py [hunk 1]
```diff
@@ -12,12 +12,19 @@ class InfernoCop(discord.Client):
     def __init__(self, config, **options):
         super().__init__(**options)
         self.config = config
+        self.cegan_username = "Praise the Morning Lord"
         self.counter_clayton = 0
         self.counter_patrick = 0
+        selt.counter_cegan = 0
```
Comment (evallen) [inline inferno_cop.py:18]: Should be self.counter_cegan

**Review by evallen — CHANGES_REQUESTED** [review]
Typo in self.counter_cegan

**Comment by johnsnoa** [comment]
Once I get home, I'll take care of it

**Pull request closed without merging**